swan-cern/Dockerfile

```diff
@@ -78,15 +67,18 @@ ADD config/dask-labextension.yaml /etc/dask/labextension.yaml
 USER root
 
 # Dependency of swandaskcluster
-RUN ln -s $(pip show swanportallocator | grep -oP 'Location: \K.*')/swanportallocator ${DASK_LIB_DIR}
+RUN ln -s $(pip show swanportallocator | grep -oP 'Location: \K.*')/swanportallocator ${SWAN_LIB_DIR}/nb_term_lib
```

Comment:
So, we do need to expose this in the permanent path?

Can you clarify when this is needed? It's because is needed in the dask proxy? But, wouldn't that work with the temp dir instead?

Reply:
As discussed, it's needed as an import dependency of swandaskcluster. No port request is done by the notebook kernel process, but such process does import swandaskcluster to get a Dask Security object when a Dask client is created.